dvc/repo/tree.py

```diff
@@ -55,7 +55,9 @@ def _get_granular_checksum(self, path, out, remote=None):
                 return entry[out.remote.tree.PARAM_CHECKSUM]
         raise FileNotFoundError
 
-    def open(self, path, mode="r", encoding="utf-8", remote=None):
+    def open(
+        self, path, mode="r", encoding="utf-8", remote=None
+    ):  # pylint: disable=arguments-differ
```

Comment:
Should we start preferring the use of **kwargs in these kinds of cases instead of suppressing this pylint test each time?